.github/workflows/template-ui-tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy - UI Tests
+
+on:
+  workflow_call:
+    inputs:
+      deploy_url:
+        type: string
+        required: true
+
+jobs:
+  test:
+    name: Run Playwright tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Install Playwright Browsers
+        run: yarn playwright install --with-deps
```

Comment:
this takes a while to run

can we install only the dependencies required to run the tests
browsers - can we use the playwright docker image to avoid the install?